content/docs/rfcs/31/README.md

```diff
@@ -118,8 +118,42 @@ Supported key type is `secp256k1` only.
 
 In the future, an extension of this RFC could be made to support other elliptic curve cryptography such as `ed25519`.
 
+# `waku2` ENR key
+
+We propose the definition of a `waku2` field key:
```

Comment:
Language like "propose" seems odd to me. Is this how we define it or not?

Reply:
Right. "We define a `waku2` field key" is clearer and more spec-like, though I tried to be consistent with [similar paragraph in this spec](https://github.com/vacp2p/rfc/pull/472/files#diff-6ecda219ddbedfc97347e30b248c8c8e0b7c7b45cfa06c9f1b0c85a122595ce9R54). I'd rather update both, though.

Reply:
Done in 27bd3a8.